python/ray/data/tests/test_huggingface.py

```diff
@@ -18,21 +20,26 @@ def test_from_huggingface(ray_start_regular_shared):
     ):
         ray.data.from_huggingface(data)
 
-    ray_datasets = {
-        "train": ray.data.from_huggingface(data["train"]),
-        "validation": ray.data.from_huggingface(data["validation"]),
-        "test": ray.data.from_huggingface(data["test"]),
-    }
+    for num_par in [1, 4]:
```

Comment:
can we parametrize these with pytest parameters? https://docs.pytest.org/en/7.1.x/example/parametrize.html